modules/ztp-post-install-spoke-factory-pipeline-checks.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="post-spoke-factory-pipeline-verification-checks_{context}"]
+= Post edge cluster factory pipeline verification checks
+
+Perform the following steps after completion of the edge cluster factory pipeline run.k
```

Comment:
Should the period be after run and then the k? I'm confused if the k is maybe an error?